### PR TITLE
[CI/CD] Install the latest still safe to display cyrillic texts version of pango for Windows nightly

### PR DESCRIPTION
**Pull request by victoryforce**

### CODE REVIEW

**Comment by kmilos** [comment]
Wasn't this never fixed upstream?!?!

**Comment by wpferguson** [comment]
I'm still using pango 1.50.11-1.  I had @victoryforce check a couple of months ago and whatever was current still wasn't working.

**Comment by victoryforce** [comment]
@kmilos And what's current as of today is still not working, of course I checked that before creating the PR.

**Comment by kmilos** [comment]
Hm, this says closed, but didn't get into details: https://gitlab.gnome.org/GNOME/pango/-/issues/718

**Comment by kmilos** [comment]
In fact, I'd rather we keep catching issues in nightlies rather than trying to avoid them.

**Comment by victoryforce** [comment]
> In fact, I'd rather we keep catching issues in nightlies rather than trying to avoid them.

Please clarify. I don't get your idea. Nightly builds should deliver a quality built program. Monitoring this issue status is my job, not the nightly user's. They have a lot more to report.

File: .github/workflows/nightly.yml [hunk 1]
```diff
@@ -239,6 +239,10 @@ jobs:
         run: |
           # It's a temporary fix, to be removed when the issue will be resolved
           pacman -U --noconfirm https://repo.msys2.org/mingw/ucrt64/mingw-w64-ucrt-x86_64-exiv2-0.27.7-1-any.pkg.tar.zst
+      - name: Install the latest still safe to display cyrillic texts version of pango
```
Comment (TurboGit) [inline .github/workflows/nightly.yml:242]: I've probably missed it, what the issue? Maybe open one if not already and reference it here?

File: .github/workflows/nightly.yml [hunk 2]
```diff
@@ -239,6 +239,10 @@ jobs:
...
+        run: |
+          # It's a temporary fix, to be removed when the issue will be resolved
```
Comment (TurboGit) [inline .github/workflows/nightly.yml:244]: I've probably missed it, what the issue? Maybe open one if not already and reference it here?

Comment (victoryforce) [reply .github/workflows/nightly.yml:244]: Issue has long been open: #13084. Yes, I'll amend the comment to include a reference to the issue.

**Comment by TurboGit** [comment]
I agree with @victoryforce, let's workaround this issue now to get a better user's experience.

**Pull request merged**